meteor/client/ui/RundownView/RundownViewEventBus.ts

```diff
@@ -59,6 +62,10 @@ export interface ShelfStateEvent extends IEventContext {
 	state: boolean | 'toggle'
 }
 
+export interface MiniShelfQueueAdLibEvent extends IEventContext {
+	forward: boolean
```

Comment:
Some explanation here is needed. What does `forward` mean? What if `forward: false`? Does that mean "backward"? Also the other "moving" implementations use a `delta`, I think it would be nice to repeat that pattern here. Then it would be clear what `delta: 1` and `delta: -1` mean.